bundles/org.eclipse.core.jobs/src/org/eclipse/core/internal/jobs/JobManager.java

```diff
@@ -1100,19 +1101,17 @@ boolean join(InternalJobGroup jobGroup, long timeout, IProgressMonitor monitor)
 	}
 
 	/**
-	 * Returns a non-null progress monitor instance.  If the monitor is null,
-	 * returns the default monitor supplied by the progress provider, or a
+	 * Returns a non-null progress monitor instance. If a progress provider is
+	 * given, returns the monitor for this supplied by the progress provider, or a
```

Comment:
"is given" would fit if that would be an argument ro the method. I think "is configured" or "is set" would fit better